jenkins/run_aggregation_broker.groovy

```diff
@@ -0,0 +1,35 @@
+#!groovy
+@Library('cditma-utils') _
+def utils = new cditma.Utils(this);
+
+pipeline{
+    agent{node "aggregation_broker"}
+
+    parameters{
+        string(name: 'db_ip_address', defaultValue: "${env.DOCKER_IP_ADDRESS}", description: 'The IP address of the database')
+        //string(name: 'port', defaultValue: '12345', description: 'The port that the Environment Manager should use')
+        string(name: 'broker_endpoint', defaultValue: "${env.ENVIRONMENT_MANAGER_IP_ADDRESS}", description: 'The qualified endpoint that will be used by clients to connect to the broker')
+        string(name: 'db_password', defaultValue: '', description: 'The password for the database')
+    }
+
+    stages{
+        stage("Run Aggregation Broker"){
+            steps{
+                script{
+                    def args = "-i ${params.db_ip_address} "
+                    
+                    if(params.tao_ns_endpoint){
+                        args += "-b ${params.broker_endpoint} "
+                    }
+                    if(params.qpid_broker_endpoint){
+                        args += "-p ${params.db_password} "
+                    }
+
+                    if(!utils.runScript("/logan/bin/aggregation_broker ${args}") == 0){
```

Comment:
this would be cleaner:
utils.runScript("/logan/bin/aggregation_broker ${args}") != 0